### PR TITLE
Iturcanu

### PR DESCRIPTION
**Pull request by TurcanuIgori**
Add new files

### CODE REVIEW

**Comment by TurcanuIgori** [comment]
edit please the file index.php

**Comment by NIckyKrow** [comment]
Nicky comment

File: Denis Betian!!.txt [hunk 1]
```diff
@@ -1 +1,4 @@
-text
\ No newline at end of file
+alt text
```
Comment (TurcanuIgori) [inline Denis Betian!!.txt]: gresit

Comment (TurcanuIgori) [reply Denis Betian!!.txt]: am corectat

File: Denis Betian!!.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+text
```
Comment (NIckyKrow) [inline Denis Betian!!.txt]: Good text!

Comment (TurcanuIgori) [reply Denis Betian!!.txt]: thanks

**Pull request merged**